package/gluon-core/files/lib/gluon/upgrade/040-site-code

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/lua
+
+local sysconfig = require 'gluon.sysconfig'
+local site = require 'gluon.site_config'
+
+sysconfig.site_code = site.site_code
```

Comment:
Where is this needed/where was this done before? It seems to be totally unrelated to everything else here.